### DISCUSSION TITLE
how to add baggage to metrics using agent extension?

### DISCUSSION THREAD

**Discussion by tuhao1020** [open-telemetry/opentelemetry-java-instrumentation#9871] [Q&A]
It is possible to add baggage in spanprocessor, but I didn't find an example of adding baggage to metrics, can anyone give some code snippets?

**Comment by trask** [comment]
it's an experimental API, but check out https://github.com/open-telemetry/opentelemetry-java/blob/9ac678e81bce7c12820acdb846d22d7957b8b15f/sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/view/AttributesProcessor.java#L86
